SteamKit2/SteamKit2/Base/ClientMsg.cs

```diff
@@ -28,14 +28,15 @@ internal ClientMsgProtobuf( int payloadReserve = 0 )
         /// <value>
         /// 	<c>true</c> if this instance is protobuf backed; otherwise, <c>false</c>.
         /// </value>
-        public override bool IsProto { get { return true; } }
+        public override bool IsProto => true;
+
         /// <summary>
         /// Gets the network message type of this client message.
         /// </summary>
         /// <value>
         /// The network message type.
         /// </value>
-        public override EMsg MsgType { get { return Header.Msg; } }
+        public override EMsg MsgType => Header.Msg;

```

Comment:
I think an expression body change like this modifies the property to be both get and set, something we don't want here. Ditto for a bunch of get only properties.